src/components/overflow-menu/overflow-menu.scss

```diff
@@ -13,6 +13,16 @@ $css--plex: true !default;
   @extend .#{$prefix}--overflow-menu;
 }
 
+:host(#{$prefix}-overflow-menu)[size='sm'] {
+  width: rem(32px);
+  height: rem(32px);
```

Comment:
```suggestion
  @extend .#{$prefix}--overflow-menu--sm;
```

Reply:
@asudoh Good call! Interesting that this works, in the past I've had problems extending classes directly from other imports.